src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -207,6 +207,33 @@ void M()
             EOF();
         }
 
+        [Fact]
+        public void TestInterpolatedStringWithNewLinesInExpression()
+        {
+            var text = @"$""Text with {\nnew[] {\n 1, 2, 3 \n}[2]\n}parts and new line expressions!""";
+            text = text.Replace(@"\n", System.Environment.NewLine);
```

Comment:
I don't see the need for this replacement call

Reply:
it is a @" string - without escapes...

Reply:
gah  even more confusing.  I hate it :)